app/src/containers/ConnectedSidebar/tests/index.test.tsx

```diff
@@ -78,6 +118,75 @@ describe('components/ConnectedSidebar', () => {
         </MemoryRouter>
       </Provider>
     );
+
+    const submenu = wrapper
+      .find('SubMenu[title="Admin"]')
+      .first()
+      .last()
+      .prop('children') as ReactWrapper[];
+
+    expect(submenu[3].key).toMatch('location');
+  });
+
+  it('displays menu links for enabled Form Configuration module', () => {
+    const envModule = require('../../../configs/env');
+    envModule.ENABLE_FORM_CONFIGURATION = 'true';
+
+    const wrapper = mount(
+      <Provider store={store}>
+        <MemoryRouter>
+          <ConnectedSidebar />
+        </MemoryRouter>
+      </Provider>
+    );
+
+    const submenu = wrapper
+      .find('SubMenu[title="Admin"]')
+      .first()
+      .last()
+      .prop('children') as ReactWrapper[];
+
+    expect(submenu[4].key).toMatch('form-config');
+  });
+
+  it('correctly expand users menu', () => {
+    const wrapper = mount(
+      <Provider store={store}>
+        <MemoryRouter initialEntries={[{ pathname: `/admin`, hash: '', search: '', state: {} }]}>
+          <ConnectedSidebar />
+        </MemoryRouter>
+      </Provider>
+    );
+    expect(wrapper.find('Menu').at(0).prop('children')).toMatchSnapshot();
+    wrapper.unmount();
+  });
+
+  it('correctly expand location menu', () => {
+    const envModule = require('../../../configs/env');
+    envModule.ENABLE_LOCATIONS = 'true';
+
+    const wrapper = mount(
+      <Provider store={store}>
+        <MemoryRouter initialEntries={[{ pathname: `/location`, hash: '', search: '', state: {} }]}>
+          <ConnectedSidebar />
+        </MemoryRouter>
+      </Provider>
+    );
+    expect(wrapper.find('Menu').at(0).prop('children')).toMatchSnapshot();
+    wrapper.unmount();
+  });
+
+  it('correctly expand form-config menu', () => {
```

Comment:
@ashfahan Do we need to add this test case?. It was catered for in `Test order of menu` test case